app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  after_create :update_post_likes_counter
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_like_counter` should be marked as _private_ after implementing the `after_create` method.